cmd/agent/app/integrations.go

```diff
@@ -608,3 +620,33 @@ func freeze(cmd *cobra.Command, args []string) error {
 
 	return tuf(tufArgs)
 }
+
+func integrationVersionCmdRun(cmd *cobra.Command, args []string) error {
+
+	cachePath, err := getTUFPipCachePath()
+	if err != nil {
+		return err
+	}
+
+	packageName := strings.Replace(args[0], "_", "-", -1)
+
+	version, err := installedVersion(packageName, cachePath)
+	if err != nil {
+		return fmt.Errorf("could not get current version of %s: %v", packageName, err)
+	}
+
+	if version == nil {
+		// Package not installed, return 0 and print nothing
+		return nil
+	}
+
+	if quietVersion {
+		// Print only the version for easier parsing
+		fmt.Println(version)
+	} else {
+		fmt.Printf("Installed version of %s: %s", packageName, version)
```

Comment:
You could just add a `\n` at the end of the first argument and remove the next line.